### DISCUSSION TITLE
Reusing username.

### DISCUSSION THREAD

**Discussion by hello-users9** [community/community#143654] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! I'm trying to create an organization with a username that was used on a deleted account and I get the error: Organization name is unavailable.

How long does it take before I can reuse a username that is no longer taken?

**Comment by Annapoornaaradhya** [comment, marked as answer]
@hello-users9 If the account using the username was deleted, GitHub doesn’t immediately release the username for reuse. Generally, GitHub may retain the username for an indefinite period, and the exact timing for its availability is not specified. In some cases, usernames may not be released at all.

If you need the username urgently, consider reaching out to GitHub Support to check the status or confirm if it may become available in the future. Alternatively, you could try variations of the username for your organization.

**Reply by hello-users9** [reply]
Please tell me the right form for communication? support@github.com doesn't accept letters

**Reply by Annapoornaaradhya** [reply]
@hello-users9 https://support.github.com/request this may help you check it out once

**Comment by hello-users9** [comment]
<img width="1438" alt="image" src="https://github.com/user-attachments/assets/221648b4-1ea3-4ce7-9522-6b31497a82a7">

**Reply by Annapoornaaradhya** [reply]
If this is the case then maybe you cannot reuse the username try naming it with slight variants so that it does not much difference
